packages/teams-js/src/internal/communication.ts

```diff
@@ -344,7 +349,7 @@ export function sendMessageToParent(
   if (callback) {
     // eslint-disable-next-line @typescript-eslint/ban-ts-comment
     // @ts-ignore
```

Comment:
This causes issues now that I've changed callback to be of type `((value?: unknown) => void)`

Reply:
While I believe it is correct to have removed the `Function` type usage from the `promiseCallbacks`, I don't think you can remove it from the `callbacks` map because it's being populated from this function, and this function takes `callback` as a parameter that's typed as `Function` -- so the `callbacks` map where it is stored has to stay as `Function` (unless you want to also change the parameter type on this function and keep walking up the chain for all callers of this function and so on -- a way more invasive change than I think is appropriate to do in this PR).

In summary: I think you have to leave `callbacks` as a map to `Function`

And then maybe you can remove these suppressions. But if not, that's okay, too.